examples/gallery/embellishments/text_formatting.py

```diff
@@ -0,0 +1,41 @@
+"""
+Text formatting
+===============
+
+There are various options to format text added to a plot as well as the title of the
```

Comment:
Maybe rephrase it to something like:

> There are various options to format text strings added to a plot, e.g., 
> text added via `Figure.text`, title of a plot, labels of colorbars, 
> Cartesian axes, and legend entries,

Reply:
See commit https://github.com/GenericMappingTools/pygmt/pull/3987/commits/2462e992132a0ccab2858fb9749bde18ebd86c53.